text_generation/causal_lm/cpp/continuous_batching/library/src/sampler.hpp

```diff
@@ -395,6 +395,7 @@ SamplerOutput Sampler::sample(std::vector<SequenceGroup::Ptr> & sequence_groups,
         size_t actual_seq_len = sequence_group->get_num_scheduled_tokens(); // points to a token which needs to be sampled
         size_t padded_amount_of_processed_tokens = std::max(actual_seq_len, batch_seq_len);
         const GenerationConfig& sampling_params = sequence_group->get_sampling_parameters();
+        OPENVINO_ASSERT(sampling_params.min_new_tokens <= sampling_params.max_new_tokens);
```

Comment:
maybe we can introduce `validate()` method to generation config and will call it once user has added request to our engine?

Reply:
in such `validate` we can also add that `n > 1` is valid for beam search or multinomial only, because it does not make sense for greedy